### PR TITLE
doc(notebook): update readme and standalone quickstart doc, notebooks

### PR DESCRIPTION
**Pull request by tianweidut**
## Description
- depend on: https://github.com/star-whale/starwhale/pull/1841
- colab example: https://colab.research.google.com/drive/1eodNQUMOCC5lsQI-5QJH5A6jGlrJWHoS?usp=sharing

## Modules
- [x] Others

## Checklist
- [x] run code format and lint check
- [ ] add unit test
- [x] add necessary doc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -285,10 +275,10 @@ Let's go ahead and finish the tutorial on the on-premises instance.
     ```
 
     > For users in the mainland of China, please add the startup parameter：`--image-mirror-country=cn`.
-
-    ```bash
-    minikube start --image-mirror-country=cn
-    ```
+    >
+    > ```bash
+    > minikube start --image-mirror-country=cn
```
Comment (jialeicui) [inline README.md:280]: `--image-mirror-country=cn` may not be enough
Specify the k8s version like 1.23.8 ?

ref:
- https://github.com/kubernetes/minikube/issues/15451#issuecomment-1409817778
- https://github.com/kubernetes/minikube/issues/15270#issuecomment-1304966090

File: docs/docs/quickstart/standalone.md [hunk 2]
```diff
@@ -30,44 +34,67 @@ git clone https://github.com/star-whale/starwhale.git
...
 ## 4. Building Model
 
 Model example code are in the `example/mnist` directory.
 
-- Build a Starwhale Model:
+- Download pre-trained model file:
+
+  ```bash
+  cd example/mnist
+  make download-model
```
Comment (jialeicui) [inline docs/docs/quickstart/standalone.md:87]: Add comments for users in mainland of China?  
```sh
CN=1 make download-model
```

File: docs/i18n/zh/docusaurus-plugin-content-docs/current/quickstart/standalone.md [hunk 3]
```diff
@@ -56,14 +69,28 @@ Runtime的示例程序在 `example/runtime/pytorch` 目录中。
...
 ## 4. 构建Starwhale Model模型包
 
 Model的示例程序在 `example/mnist` 目录中。
 
-- 构建Starwhale Model：
+- 下载预先训练好的模型文件：
+
+  ```bash
+  cd example/mnist
+  make download-model
```
Comment (jialeicui) [inline docs/i18n/zh/docusaurus-plugin-content-docs/current/quickstart/standalone.md:86]: Better add `CN=1`.

Comment (tianweidut) [reply docs/i18n/zh/docusaurus-plugin-content-docs/current/quickstart/standalone.md:86]: ok

File: docs/docs/quickstart/standalone.md (hunk 2 above)
Comment (tianweidut) [reply docs/docs/quickstart/standalone.md:87]: ok

File: README.md [hunk 4]
```diff
@@ -103,171 +103,161 @@ Starwhale is an MLOps platform. It provides **Instance**, **Project**, **Runtime
...
+    > For users in the mainland of China, the following conf file is an option:
+    >
+    > ```conf
+    > [global]
+    > cache-dir = ~/.cache/pip
+    > index-url = https://mirrors.aliyun.com/pypi/simple/
+    > extra-index-url = https://pypi.doubanio.com/simple
+    >
+    > [install]
+    > trusted-host=pypi.doubanio.com mirrors.aliyun.com
```
[the remark this answers is not in the document]
Comment (tianweidut) [reply README.md:122]: ok

File: README.md (hunk 1 above)
Comment (tianweidut) [reply README.md]: done

File: example/notebooks/quickstart-standalone.ipynb [hunk 5]
```diff
@@ -0,0 +1,361 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "swcli - eval run --model mnist/version/latest --dataset mnist/version/latest --runtime pytorch/version/latest"
```
Comment (jialeicui) [inline example/notebooks/quickstart-standalone.ipynb:310]: remove the `-`?

Comment (tianweidut) [reply example/notebooks/quickstart-standalone.ipynb]: done

**Review by jialeicui — APPROVED** [review]
LGTM

**Pull request merged**